src/libraries/System.Diagnostics.TraceSource/src/System/Diagnostics/SwitchAttribute.cs

```diff
@@ -12,8 +12,8 @@ namespace System.Diagnostics
                     AttributeTargets.Event | AttributeTargets.Method | AttributeTargets.Property)]
     public sealed class SwitchAttribute : Attribute
     {
-        private Type _type;
-        private string _name;
+        private Type _type = null!; // Initialized using property
+        private string _name = null!;
```

Comment:
Same here, could assign to fields directly to remove need for forcing.

Reply:
Property setter doing some logic (null check and length check) so I preferred to leave that as is, let me know if we want to add all those logic into constructor